logger/transport.go

```diff
@@ -98,6 +98,12 @@ func (t *transport) send(msgs []Message) error {
 			Level: msg.Options.Level,
 		}
 
+		timestamp := msg.Options.Timestamp
+		if timestamp.IsZero() {
+			timestamp = time.Now()
+		}
+		line.Timestamp = timestamp.UnixNano() / int64(time.Millisecond)
```

Comment:
@dm36, can you explain this operation here and why you do that?

Reply:
https://stackoverflow.com/questions/24122821/go-golang-time-now-unixnano-convert-to-milliseconds